src/Utilities/CacheKeys.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using Microsoft.Extensions.Caching.Memory;
+
+namespace ePiggyWeb.Utilities
+{
+    public static class CacheKeys
```

Comment:
Could you explain what this class does?  in short

Reply:
It's only for more convenient use of the cache - the cache keys are saved there so it's easier to change them all in bulk if needed, and I put a method for default options for currency because they're always the same